### PR TITLE
Removed unused messages

### PR DESCRIPTION
**Pull request by podhrmic**
Removed message that don't appear in the code base (key: grep -r "MSG_NAME" sw/*)

Added comments for possibly unused messages (i.e. they appear in some codebase, but not in sw/simulator nor sw/airborne).

Follow up on #41 and #27

### CODE REVIEW

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -706,25 +592,8 @@
       <field name="index_list" type="uint8[]"/>
     </message>
 
-    <message name="CROSS_TRACK_ERROR" id="80">
-      <field name="cross_track_error" type="float" unit="m"/>
-      <field name="cte_int" type="float"/>
-    </message>
-
-    <message name="GENERIC_COM" id="81">
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: Even if not really used at the moment, it is still part of the ground side and it corresponds to a message sent over satcom link.

File: message_definitions/v1.0/messages.xml [hunk 2]
```diff
@@ -1984,6 +1614,7 @@
       <field name="humidity" type="float" unit="%"/>
     </message>
 
+  <!-- Do we need this? Is the turntable still a thing? -->
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: we still have one at ENAC

File: message_definitions/v1.0/messages.xml [hunk 3]
```diff
@@ -2018,16 +1649,15 @@
       <field name="sonar" type="float"> measurement from the sonar (mm)</field>
     </message>
 
-    <message name="VIDEO_SYNC" id="225">
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: This is used in `sw/ground_segment/misc/video_synchronizer` tool

File: message_definitions/v1.0/messages.xml (hunk 1 above)
Comment (podhrmic) [inline message_definitions/v1.0/messages.xml]: Updated,

File: message_definitions/v1.0/messages.xml (hunk 2 above)
Comment (podhrmic) [inline message_definitions/v1.0/messages.xml]: OK

File: message_definitions/v1.0/messages.xml (hunk 3 above)
Comment (podhrmic) [inline message_definitions/v1.0/messages.xml]: OK

**Pull request merged**